flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/CodeGeneratorContext.scala

```diff
@@ -143,6 +148,9 @@ class CodeGeneratorContext(val tableConfig: TableConfig) {
     reusableLocalVariableStatements(methodName) = mutable.LinkedHashSet[String]()
   }
 
+  def setSplit(): Unit = {
```

Comment:
`enableCodeSplit()` or `setCodeSplit()`?  Please also add Javadoc above this method.